src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -303,7 +331,7 @@ def quantize_input(self, *inputs, config, op, op_type, op_name):
         raise NotImplementedError('Quantizer must overload quantize_input()')
 
 
-    def _instrument_layer(self, layer, config):
+    def _wrap_modules(self, layer, config):
```

Comment:
`_wrap_modules` is not a proper name for Quantizer,  and it need to return a wrapper

Reply:
Quantizer needs to take same changes like Pruner to enable dataparallel.